build.sbt

```diff
@@ -202,7 +202,7 @@ lazy val cdfdump = (project in file("cdf_dump"))
 
 lazy val fatJar = project.settings(
   commonSettings,
-  name := "cdp-spark-datasource-fat",
+  name := "cdf-spark-datasource-fat",
```

Comment:
If we don't publish a non-fat version, we should consider removing the `-fat` suffix, but that can be done in another PR.